_data/changelogs/component-header.yml

```
@@ -3,9 +3,10 @@ type: component
 changelogURL:
 items:
   - date: 2023-03-13
-    summary: Logo text width setting now works as expected.
-    summaryAdditional: Now the design system respects the value passed to `$theme-header-logo-text-width`.
-    isBreaking: false
+    summary: Extended headers now respect the value passed to `$theme-header-logo-text-width`.
+    summaryAdditional: For extended headers, the default setting value is narrower than
+      the previous hard-coded value, which can cause unexpected line breaks.
```

Would it be a good idea to specify the linebreak occurs within `usa-navbar` or on logo text? or do you think it'd be redundant with the setting name? Just considering where people might look to resolve this with CSS

```suggestion
    summaryAdditional: For extended headers, the default setting value is narrower than
      the previous hard-coded value, which can cause unexpected line breaks for `usa-logo__text`.
```

---

Good suggestion! Added in [e1d5aad](https://github.com/uswds/uswds-site/pull/2053/commits/e1d5aadb77ce280eb683ae9ad4b22cdaba18adc7)